src/Sylius/Component/Core/Repository/ProductReviewRepositoryInterface.php

```diff
@@ -38,4 +40,20 @@ public function findLatestByProductId($productId, int $count): array;
      * @return array|ReviewInterface[]
      */
     public function findAcceptedByProductSlugAndChannel(string $slug, string $locale, ChannelInterface $channel): array;
+
+    /**
+     * @param string $locale
+     * @param string $productCode
+     *
+     * @return QueryBuilder
+     */
+    public function createQueryBuilderByProductCode(string $locale, string $productCode): QueryBuilder;
```

Comment:
Adding methods to an interface is a BC break.

Reply:
@pamil, in this case how can proceed next? Should we put the methods only into the ProductReviewRepository?

Reply:
According to our proposed BC promise (#8901), it isn't a BC break 🎉